profile.md

```diff
@@ -37,7 +37,16 @@ _Authored by the WLCG AuthZ Working Group_
 
 # Introduction
 
-This document describes how WLCG users may use the available geographically distributed resources without X.509 credentials.  In this model, clients are issued with bearer tokens; these tokens are subsequently used to interact with resources.  The tokens may contain authorization groups and/or capabilities, according to the preference of the VO, applications and relying parties. 
+This document describes how WLCG users may use the available geographically distributed resources without X.509 credentials.  In this model, clients are issued with bearer tokens; these tokens are subsequently used to interact with resources.  The tokens may contain authorization groups and/or capabilities, according to the preference of the virtual organisation, applications and relying parties.
+
+A virtual organisation (VO) is a group of users that are
+self-managing, often with internal structure (such as groups and
```

Comment:
```suggestion
independently managed, often with internal structure (such as groups and
```
At Fermilab we have a group that provides services to many VOs (both those that we consider sub-VOs and full VOs), including the management of computing infrastructures.  So `self-managed` isn't accurate for them.

Reply:
Thanks.